specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2020-06-14/examples/KustoAttachedDatabaseConfigurationsCreateOrUpdate.json

```diff
@@ -1,61 +1,61 @@
-{
-  "parameters": {
-    "subscriptionId": "12345678-1234-1234-1234-123456789098",

```

Comment:
Looks like some git merge issue or formatting change?
Why is this file showing up - This should not be changed.

Reply:
There was some styling/formatting error (Prettier check) and the suggestion  was to use npm run prettier-fix
I think it fixed the formatting issues in all the files